index.html

```diff
@@ -38,9 +40,9 @@ <h2>Security</h2>
       </ul>
     </div>
 
-    <div class="index-column" style="margin-top: 0rem;">
+    <div class="index-column" id="right-index-column">
       <h2>Privacy</h2>
-      <ul style="margin-left: -1rem;">
+      <ul id="index-column-list">
```

Comment:
I think `index-column-list` should be a class since IDs should be unique on a page.